css/styles.css

```diff
@@ -0,0 +1,240 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", Helvetica, Arial, Verdana, sans-serif;
+}
+
+.space-b {
+  justify-content: space-between;
+}
+
+.absolute {
+  position: absolute;
+}
+
+.font-blue {
+  color: #08c;
+  text-decoration: none;
+}
+
+.w-1200px {
+  width: 100%;
+  min-width: 1200px;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.footer-nav ul .flex-item {
+  border-right: 1px solid rgb(155, 145, 145);
+  justify-content: center;
+  padding-right: 5px;
+}
+
+.main-nav {
+  z-index: 10;
+  top: 75px;
+  right: calc((100% - 900px) / 2);
+  width: 955px;
+  background: linear-gradient(
```

Comment:
- [ ] You have linter errors here. It is probably the format on save. Usually, when I do complex gradients like this I left it disabled.

  44:13  ✖  Expected newline after ":" with a multi-line declaration   declaration-colon-newline-after